src/daft-session/src/options.rs

```diff
@@ -2,17 +2,9 @@
 pub(crate) const _DAFT_SESSION: &str = "default";
 pub(crate) const _DAFT_SESSION_USER: &str = "daft";
 pub(crate) const _DAFT_SESSION_TEMP_DIR: &str = "/tmp";
-pub(crate) const DAFT_SESSION_DEFAULT_CATALOG: &str = "daft";
 
-#[derive(Debug)]
+#[derive(Debug, Default)]
 pub(crate) struct Options {
-    pub curr_catalog: String,
-}
-
-impl Default for Options {
-    fn default() -> Self {
-        Self {
-            curr_catalog: DAFT_SESSION_DEFAULT_CATALOG.to_string(),
-        }
-    }
+    pub curr_catalog: Option<String>,
+    pub curr_namespace: Option<Vec<String>>,
```

Comment:
I see we're moving away from the term "namespace" and into "qualifier" in some places but not in others. How are you thinking about the difference between these two terms?

Reply:
* Here's the PR for qualified identifiers in SQL https://github.com/Eventual-Inc/Daft/pull/3860
* Qualifies are the first parts of an identifier
* Namespaces are a path within a catalog.

The qualifier can contain a catalog, so it's not exactly a namespace. Namespace is like a dir in a volume if the volume is catalog.